opentelemetry/proto/collector/logs/v1/logs_service.proto

```diff
@@ -43,4 +43,32 @@ message ExportLogsServiceRequest {
 }
 
 message ExportLogsServiceResponse {
+  // The details of a partially successful export request.
+  //
+  // If the request is only partially accepted
+  // (i.e. when the server accepts only parts of the data and rejects the rest)
+  // the server MUST initialize the `partial_success` field and MUST
+  // set the `rejected_log_records` with the number of log records it rejected.
+  //
+  // The server MUST NOT set the `partial_success` field when the
+  // request was accepted in its entirety.
+  ExportLogsPartialSuccess partial_success = 1;
+}
+
+message ExportLogsPartialSuccess {
+  // The number of rejected log records.
+  // 
+  // Given the server MUST only initialize the `partial_success` field
+  // when it rejects parts of the data, a `rejected_log_records` with
+  // value = '0' is considered an invalid result. Senders MUST
+  // interpret such responses from a server as if all log records were accepted.
```

Comment:
I would prefer to discuss this detail in https://github.com/open-telemetry/opentelemetry-specification/pull/2696#discussion_r930249217

Reply:
As discussed in the thread linked by @jmacd , we will allow `rejected_<signal>` = `0` so the server can use a `partial_success` to convey warnings/suggestions to clients via the `error_message` field. I adapted the text around it to match the spec change in [3461d1f](https://github.com/open-telemetry/opentelemetry-proto/pull/414/commits/3461d1f15a4a12b0bab5c567f22b0030b72e070e)